CONTRIBUTING.md

```diff
@@ -74,6 +74,39 @@ accepting contributions.
    you like, especially if we can't incorporate your recommendations. But please also honor
    our [Governance Rules](GOVERNANCE.md).
 
+## Branching and Workflow
+
+We use a centralized approach that leaves `main` as the bleeding edge and has version branches for bug and feature
```

Comment:
bleeding edge -> current version

Reply:
I think "bleeding edge" is more appropriate because the current version implies the most current release, which it's not.

I'll change it in compromise to "most stable dev version".